src/core/services/LineFactoryService.ts

```diff
@@ -53,19 +53,11 @@ export class LineFactoryServiceImpl {
 
   public async deployEscrow(
     contractAddress: Address,
-    minCRatio: BigNumber,
-    oracle: string,
     owner: string,
     borrower: string,
     dryRun: boolean
   ): Promise<TransactionResponse | PopulatedTransaction> {
-    return await this.executeContractMethod(
-      contractAddress,
-      'deployEscrow',
-      [minCRatio, oracle, owner, borrower],
-      'goerli',
-      dryRun
-    );
+    return await this.executeContractMethod(contractAddress, 'deployEscrow', [owner, borrower], 'goerli', dryRun);
```

Comment:
We can pass down the network from the modal or lines.actions, you can add a    { wallet } = extra.service and then use the getNetworkSelector